src/coingecko/Coingecko.ts

```diff
@@ -33,7 +33,12 @@ export class Coingecko {
     return this.instance;
   }
 
-  private constructor(private readonly host = "https://api.coingecko.com/api/v3") {}
+  public static getPro(apiKey?: string) {
+    if (!this.instance) this.instance = new Coingecko("https://pro-api.coingecko.com/api/v3", apiKey);
+    return this.instance;
+  }
```

Comment:
nit: can we just drop the host param from the get method and combine get and getPro?

Reply:
How would that work?

Reply:
Like this? https://github.com/across-protocol/sdk-v2/pull/70/commits/6b6fa44da99abf43859950696a68d284c794af59

Reply:
Yep, that's what I was thinking!